docker-compose.yml

```diff
@@ -87,6 +87,14 @@ services:
     working_dir: /usr/src/app/frontend_build/
     entrypoint: ../devops/deps_ok_then ./node_modules/.bin/grunt
 
+  npm:
+    image: node:6
+    volumes:
+      - $PWD:/usr/src/app
+      - npm_libs:/usr/src/app/frontend_build/node_modules/
+    working_dir: /usr/src/app/frontend_build/
+    entrypoint: ../devops/deps_ok_then npm
```

Comment:
NBD, but one potential improvement would be to remove the duplication with YAML references. It'd look something like:
```yml
grunt: &NODE
    image: node:6
   ...
npm:
    <<: *NODE
    entrypoint: ../devops/deps_ok_then npm
```

Reply:
This PR is sitting here because I want to implement this before merging. If you think I should just merge, lmk.